includes/JumpstartSitesEngineering.php

```diff
@@ -159,4 +185,132 @@ public function install_private_pages_section_menu_items(&$install_state) {
     $time_diff = time() - $time;
     drush_log('JSE - Finished creating Private Pages menu items: ' . $time_diff . ' seconds' , 'ok');
   }
- }
+  /**
+   * Enable a number of the home page layouts and set one to default on.
+   * @param  [type] $install_state [description]
+   * @return [type]                [description]
+   */
+  public function configure_pical_homepage_layouts (&$install_state) {
+    $time = time();
+    drush_log('JSE - Configuring PICAL homepage layouts.' . $time, 'ok');
+
+
+    $default = 'stanford_jumpstart_home_morris';
+    variable_set('stanford_jumpstart_home_active_body_class', 'stanford-jumpstart-home-morris');
+
+    $context_status = variable_get('context_status', array());
+    $homecontexts = stanford_jumpstart_home_context_default_contexts();
+
+    $names = array_keys($homecontexts);
+
+    // Enable these for use by site owners
+    $enabled['stanford_jumpstart_home_hoover'] = 1;
+    $enabled['stanford_jumpstart_home_morris'] = 1;
+    $enabled['stanford_jumpstart_home_terman'] = 1;
+    $enabled['stanford_jumpstart_home_petit'] = 1;
+
+    // Todo: remove these JSA
+    $enabled['stanford_jumpstart_home_lomita'] = 1;
+    $enabled['stanford_jumpstart_home_mayfield_news_events'] = 1;
+    $enabled['stanford_jumpstart_home_palm_news_events'] = 1;
+    $enabled['stanford_jumpstart_home_panama_news_events'] = 1;
+    $enabled['stanford_jumpstart_home_serra_news_events'] = 1;
+
+    unset($context_status['']);
+
+    foreach ($names as $context_name) {
+      $context_status[$context_name] = TRUE;
+      $settings = variable_get('sjh_' . $context_name, array());
+      $settings['site_admin'] = isset($enabled[$context_name]);
+      variable_set('sjh_' . $context_name, $settings);
+    }
+
+   // $context_status[$default] = FALSE;
+    unset($context_status['']);
```

Comment:
I shamelessly borrowed the code from here, and assumed it was needed:
https://github.com/SU-SWS/stanford_sites_jumpstart_academic/blob/7.x-4.x/includes/JumpstartSitesAcademic.php#L784